client/src/app/pages/waves/components/create-edit-wave-modal/wave-form.tsx

```diff
@@ -28,8 +70,32 @@ export const WaveForm: React.FC<WaveFormProps> = ({
 }) => {
   const { t } = useTranslation();
 
-  const waves: Wave[] = []; // TODO use the useFetchWaves query here
+  const { waves } = useFetchMigrationWaves();
   const isLoading = false; // TODO
+  const { pushNotification } = React.useContext(NotificationsContext);
+  //
+  const { stakeholders } = useFetchStakeholders();
+  const { stakeholderGroups } = useFetchStakeholderGroups();
+
+  const onCreateMigrationWaveSuccess = (data: MigrationWave) => {
+    pushNotification({
+      title: "MigrationWave",
+      message: `Migration wave created`,
```

Comment:
We're supposed to use i18n.

Reply:
I tried to use the `toastr.success.added` but it kept failing on `response.data.name`. I went looking to see what I could be doing wrong with the query but couldn't find it. Any ideas?